Modules/Forum/classes/class.ilForumTopic.php

```diff
@@ -565,40 +565,38 @@ public function movePosts($old_obj_id, $old_pk, $new_obj_id, $new_pk)
 				}
 			}
 
-			$this->db->lockTables(
-				array(
-					0 => array('name' => 'frm_user_read',     'type' => ilDBConstants::LOCK_WRITE),
-					1 => array('name' => 'frm_thread_access', 'type' => ilDBConstants::LOCK_WRITE)
-				)
-			);
-
-			$this->db->manipulateF('
+			$ilAtomQuery = new ilAtomQuery($ilDB);
```

Comment:
This is an example how to replace current table locks with an abstraction for galera support